Mixtrack Pro 3 controller - Autoloop adjustments

djsteph opened the pull request:
Adjustments to mapping of Autoloop pads to change default values from "1/8, 1/4, 1/2 and 1" beat loop to "1,2,4 and 8" beat loops; same as Serato and VDJ 

I changed 2 files by uploading them directly to Github...Sorry, this will make comparing changes harder. Changes have been tested by me with my MTP3 Pro controller

Steph

djsteph committed:
Mixtrack Pro 3 controller - Autoloop adjustments

Adjustments to mapping of Autoloop pads to change default values from 1/8, 1/4, 1/2 and 1 beat loop to 1,2,4 and 8 beat loops

sblaisot commented:
Please update windows uninstaller to also remove these 2 new files:
http://mixxx.org/wiki/doku.php/contributing_mappings#windows_installer_update

sblaisot commented:
Oh, sorry, these are not new files, these are files that are not in the right directory. you uploaded them at the root of the project. see https://github.com/djsteph/mixxx/tree/Mixtrack-Pro-3-v1.4
you should update them in the res/controllers subdirectory

sblaisot commented:
you just have to `git mv -f Numark* src/controllers` then `git commit -m "commit message"` and then `git push` to move them in the right place

djsteph committed:
Delete Numark-Mixtrack-3-scripts.js

Removed file updloaded by error on root

djsteph committed:
Delete Numark-Mixtrack-3.midi.xml

Delete file uploaded by error on root

djsteph committed:
Mixtrack Pro 3 controller - Autoloop adjustments

replaced Numark-Mixtrack-3.midi.xml and Numark-Mixtrack-3-scripts.js with new files containing adjusted code for Autoloop pads

djsteph committed:
Changed version number in Mixtrack 3 Pro XML File

changed version number

sblaisot commented on res/controllers/Numark-Mixtrack-3.midi.xml:
```diff
@@ -1,919 +1,923 @@
+	<author>Stéphane Morin</author>
+	<description>The Numark Mixtrack 3 and Numark Mixtrack Pro 3 are the same controller except that the Pro version has an integrated sound card.</description>
+        <forums>http://mixxx.org/forums/viewtopic.php?f=7&amp;t=7286</forums>
+<<<<<<< HEAD
```
you left merge conflicts in this file

djsteph committed:
Removed merge conflict

Removed merge conflict from Numark-Mixtrack-3.midi.xml

djsteph commented:
I removed the conflict comments. Should be good now.

djsteph committed:
Padmode error correction

Correct invalid call (negate function for padmode)
removed line 1387: NumarkMixtrack3.PadModeButton = !NumarkMixtrack3.PadModeButton;
